### ISSUE TITLE
v6.0.0 transaction metadata: docs mismatch with actual behavior

### ISSUE THREAD

**Issue by mkazlauskas** [CardanoSolutions/ogmios#334]
In [Transaction Metadata](https://github.com/CardanoSolutions/ogmios/blob/master/architectural-decisions/accepted/017-api-version-6-major-rewrite.md#transactions-metadata) section of `017-api-version-6-major-rewrite.md`, it says
```
Starting from version 6, Ogmios will always return metadata as a base16 CBOR encoded object, but,
when it's possible, it will also provides a plain JSON representation of them.
```
Which is not how it currently works - it returns `cbor` **only** when either:
- metadata can't be converted to json
- Ogmios is started with `--include-metadata-cbor`

Not sure what is the intended behavior, might be a bug so decided to report it.

**Comment by KtorZ** [comment]
Right, the flag came after the documentation and that part wasn't updated. The metadata are returned in JSON indeed only when the flag is set (or when JSON isn't possible). Happy to take a PR here, or I'll fix eventually.
